composer.json

```diff
@@ -19,7 +19,8 @@
         "php": ">=5.3.9",
         "doctrine/common": "~2.3",
         "twig/twig": "~1.18",
-        "psr/log": "~1.0"
+        "psr/log": "~1.0",
+        "symfony/templating": "~2.8"
```

Comment:
@Taluu the idea is to remove the componentes un 2.8, add it as a dependency and provide a templating bundle like the security acl